framework/include/utils/MathUtils.h

```diff
@@ -27,6 +30,22 @@ Real poly2Log(Real x, Real tol, unsigned int derivative_order);
 Real poly3Log(Real x, Real tol, unsigned int derivative_order);
 Real poly4Log(Real x, Real tol, unsigned int derivative_order);
 Real taylorLog(Real x);
+Point barycentricToCartesian2D(const Point & p0,
```

Comment:
add docstring

Reply:
See euclideanMod in this header for an example

Reply:
Added docstrings in the header declaration